### PR TITLE
remove data files + add urls for nlp data

### PR DESCRIPTION
**Pull request by jerryjliu**
Offloaded NLP data/models to dropbox. A tad tedious to get links for all 25 data files for Arxiv but should be done now :)

### CODE REVIEW

File: nlp/classification/arxiv/data_urls.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+https://www.dropbox.com/s/hpag9mcd658aoa0/preds.val_5.jsonl.gz?dl=1,preds.val_5.jsonl.gz
+https://www.dropbox.com/s/65s2be0uxq94f69/preds.val_6.jsonl.gz?dl=1,preds.val_6.jsonl.gz
+https://www.dropbox.com/s/ytiwql5v47sh1yc/preds.val_7.jsonl.gz?dl=1,preds.val_7.jsonl.gz
+https://www.dropbox.com/s/8bj5giwd68hhn4i/preds.val_8.jsonl.gz?dl=1,preds.val_8.jsonl.gz
+https://www.dropbox.com/s/043cyq97ps38ddd/preds.val_9.jsonl.gz?dl=1,preds.val_9.jsonl.gz
+https://www.dropbox.com/s/gnv5mgaaxbq5zgu/train.json.gz?dl=1,train.json.gz
+https://www.dropbox.com/s/275yp649ht1xea9/val.json.gz?dl=1,val.json.gz
+https://www.dropbox.com/s/jzkjj6pptsexryv/val_with_label.json.gz?dl=1,val_with_label.json.gz
+https://www.dropbox.com/s/twhh8eu7b31nt82/val_0_with_label.json.gz?dl=1,val_0_with_label.json.gz
+https://www.dropbox.com/s/l7zbxcwew6atelw/preds.train.jsonl.gz?dl=1,preds.train.jsonl.gz
```
Comment (dhruvkedia) [inline nlp/classification/arxiv/data_urls.csv:24]: Files on line 24 and 25 weren't part of the public github repo. What are these files?

Comment (jerryjliu) [reply nlp/classification/arxiv/data_urls.csv:24]: ooh...i'm not sure actually. I think that's what train_preds.json.gz and val_preds.json.gz used to be? @asamant22 you uploaded these to dropbox

Comment (asamant22) [reply nlp/classification/arxiv/data_urls.csv:24]: Yes. These are the updated predictions for NLP that Will sent me for the model we added to ArXiv

Comment (dhruvkedia) [reply nlp/classification/arxiv/data_urls.csv:24]: looks good to me then!

**Review by dhruvkedia — APPROVED** [review]

**Review by n8cha — APPROVED** [review]

**Pull request merged**